runtime/drivers/azure/azure.go

```diff
@@ -0,0 +1,257 @@
+package azure
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/container"
+	"github.com/bmatcuk/doublestar/v4"
+	"github.com/mitchellh/mapstructure"
+	"github.com/rilldata/rill/runtime/drivers"
+	rillblob "github.com/rilldata/rill/runtime/drivers/blob"
+	"github.com/rilldata/rill/runtime/pkg/activity"
+	"github.com/rilldata/rill/runtime/pkg/fileutil"
+	"github.com/rilldata/rill/runtime/pkg/globutil"
+	"go.uber.org/zap"
+	"gocloud.dev/blob/azureblob"
+)
+
+func init() {
+	drivers.Register("azure", driver{})
+	drivers.RegisterAsConnector("azure", driver{})
+}
+
+var spec = drivers.Spec{
+	DisplayName:        "Azure Blob Storage",
+	Description:        "Connect to Azure Blob Storage.",
+	ServiceAccountDocs: "https://docs.rilldata.com/deploy/credentials/azure",
+	SourceProperties: []drivers.PropertySchema{
+		{
+			Key:         "path",
+			DisplayName: "Blob URI",
+			Description: "Path to file on the disk.",
+			Placeholder: "az://container-name/path/to/file.csv",
+			Type:        drivers.StringPropertyType,
+			Required:    true,
+			Hint:        "Glob patterns are supported",
+		},
+		{
+			Key:         "azure.storage.account",
+			DisplayName: "Azure Storage Account",
+			Description: "Azure Storage Account inferred from your local environment.",
+			Type:        drivers.InformationalPropertyType,
+			Hint:        "Set your local credentials: <code>az login</code> Click to learn more.",
+			Href:        "https://docs.rilldata.com/develop/import-data#configure-credentials-for-azure",
+		},
+	},
+	ConfigProperties: []drivers.PropertySchema{
+		{
+			Key:  "azure.storage.account",
```

Comment:
This should be `azure_storage_account`. 
Also need the other two keys: `azure_storage_sas_token` and `azure_storage_key`